zulipterminal/ui_tools/views.py

```diff
@@ -1609,7 +1609,7 @@ def __init__(
                 controller, message_links
             )
 
-            # slice_index = Number of labels before message links + 1 newlineSS
+            # slice_index = Number of labels before message links + 1 newline
```

Comment:
@Kelp710 Usually when you want to change something that you did in a previous commit, you use git interactive rebase (`git rebase -i`) and change git history such that that error never happened in the first place!

In this case you can squash your 2 commits while doing an interactive rebase and it would fix this.

Reply:
Thank you for the great advice! I`ll fit this asap